usmqe_tests/alerting/conftest.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+import time
+from usmqe.gluster import gluster
+from usmqe.usmqeconfig import UsmConfig
+import usmqe.usmssh as usmssh
+from usmqe_tests.conftest import measure_operation
+from pytest_ansible_playbook import runner
+
+# initialize usmqe logging module
+LOGGER = pytest.get_logger("pytests_test")
+pytest.set_logger(LOGGER)
+CONF = UsmConfig()
+
+
+@pytest.fixture(scope="function", autouse=True)
+def default_entities(cluster_reuse):
+    """
+    Returns basic items that are occuring in alert messages and can be used
+    for substitution.
+    """
+    cluster_identifier = cluster_reuse[
+        "short_name"] or cluster_reuse["cluster_id"]
+    node = CONF.config["usmqe"]["cluster_member"]
+    return {
+            "cluster": cluster_identifier,
+            "node": node}
```

Comment:
Just a note: this would be better implemented as a function either in the source code file of alerting tests or usmqe module. But don't change it at this point.